test/instances/savings-usds-l2/SavingsUSDSL2HyperdriveInstanceTest.t.sol

```diff
@@ -0,0 +1,215 @@
+    ) internal view override returns (uint256) {
+        return SavingsUSDSL2Conversions.convertToShares(PSM, baseAmount);
+    }
+
+    /// @dev Converts share amount to the equivalent amount in base.
+    /// @param shareAmount The share amount.
+    /// @return The converted base amount.
+    function convertToBase(
+        uint256 shareAmount
+    ) internal view override returns (uint256) {
+        return SavingsUSDSL2Conversions.convertToBase(PSM, shareAmount);
+    }
+
+    /// @dev Deploys the AerodromeLp Hyperdrive deployer coordinator contract.
+    /// @param _factory The address of the Hyperdrive factory.
+    /// @return The coordinator address.
+    function deployCoordinator(
+        address _factory
+    ) internal override returns (address) {
+        vm.startPrank(alice);
+        return
+            address(
+                new SavingsUSDSL2HyperdriveDeployerCoordinator(
+                    string.concat(config.name, "DeployerCoordinator"),
+                    _factory,
+                    address(new SavingsUSDSL2HyperdriveCoreDeployer()),
+                    address(new SavingsUSDSL2Target0Deployer()),
+                    address(new SavingsUSDSL2Target1Deployer()),
+                    address(new SavingsUSDSL2Target2Deployer()),
+                    address(new SavingsUSDSL2Target3Deployer()),
+                    address(new SavingsUSDSL2Target4Deployer()),
+                    PSM
+                )
+            );
+    }
+
+    /// @dev Fetches the total supply of the base and share tokens.
+    /// @return The total supply of base.
+    /// @return The total supply of vault shares.
+    function getSupply()
+        internal
+        view
+        virtual
+        override
+        returns (uint256, uint256)
+    {
+        return (
+            PSM.usds().balanceOf(address(PSM)),
+            convertToShares(PSM.usds().balanceOf(address(PSM)))
+        );
+    }
+
+    /// @dev Fetches the token balance information of an account.
+    /// @param account The account to query.
+    /// @return The balance of base.
+    /// @return The balance of vault shares.
+    function getTokenBalances(
+        address account
+    ) internal view override returns (uint256, uint256) {
+        return (
+            config.baseToken.balanceOf(account),
+            config.vaultSharesToken.balanceOf(account)
+        );
+    }
+
+    /// Helpers ///
+
+    /// @dev Advance time and accrue interest.
+    /// @param timeDelta The time to advance.
+    /// @param variableRate The variable rate.
+    function advanceTime(
+        uint256 timeDelta,
+        int256 variableRate
+    ) internal override {
+        IRateProvider rateProvider = IRateProvider(PSM.rateProvider());
+        uint256 chi = rateProvider.getChi();
+        uint256 rho = rateProvider.getRho();
+        uint256 ssr = rateProvider.getSSR();
+        chi = (_rpow(ssr, block.timestamp - rho) * chi) / RAY;
+
+        // console.log("chi: ", chi);
```

Comment:
Can we remove all of the commented out code?

Reply:
Absolutely, didn't mean to leave that in. Thanks for catching!